css-sizing-4/Overview.bs

```diff
@@ -573,6 +573,11 @@ Overriding Contained Intrinsic Sizes: the 'contain-intrinsic-*' properties</h3>
 			<wpt pathprefix="css/css-contain/">
 			content-visibility/content-visibility-058.html
 			</wpt>
+
+		: <dfn>from-element</dfn>
+		:: This value enables the <a>responsively-sized iframes</a>.
```

Comment:
Add a sentence here saying something like "from-element applies only to `iframe` elements, and applies regardless of the value of the `contain` CSS property". (`contain` has no impact on the `iframe` element regardless)